ledger/complete/compactor.go

```diff
@@ -45,16 +38,14 @@ type checkpointResult struct {
 // This will be resolved automaticaly after the forest LRU Cache
 // (code outside checkpointing) is replaced by something like a FIFO queue.
 type Compactor struct {
```

Comment:
We need to update the go doc here if we are moving checkpointer to somewhere else